purchase_order_general_discount/views/res_config_view.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 Tecnativa - David Vidal
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="res_config_settings_view_form_purchase" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.purchase</field>
+        <field name="model">res.config.settings</field>
+        <field name="priority" eval="25" />
+        <field
+            name="inherit_id"
+            ref="purchase.res_config_settings_view_form_purchase"
+        />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//div[@data-key='purchase']/div[hasclass('o_settings_container')][1]"
+                position="inside"
+            >
+                <div class="col-xs-12 col-6 o_setting_box">
+                    <div class="o_setting_right_pane">
+                        <label for="purchase_general_discount_field" />
+                        <span
+                            class="fa fa-lg fa-building-o"
+                            title="Values set here are company-specific."
+                            groups="base.group_multi_company"
+                        />
+                        <div class="text-muted">
+                            Select to which purchase line field will the purchase general discount be propagated.
+                        </div>
+                        <div class="content-group">
+                            <div class="mt16 row">
+                                <field
+                                    name="purchase_general_discount_field"
+                                    class="oe_inline"
+                                />
+                            </div>
+                        </div>
+                    </div>
```

Comment:
The selection for this setting is missaligned with content:
![image](https://user-images.githubusercontent.com/32061121/197957683-af2057ca-9796-462d-a1e9-affff57c2f12.png)

Could you adapt what is done for Deposit Payment to align it? Thanks 

https://github.com/OCA/purchase-workflow/blob/14.0/purchase_deposit/views/res_config_settings_views.xml#L18-L29